### DISCUSSION TITLE
Google Cloud 上でのNightscout が Chrome の拡張機能に反応しない

### DISCUSSION THREAD

**Discussion by Nobukatsuwakui** [NightscoutFoundation/xDrip#2443] [Q&A]
日本語で書きます。Google Cloud 上で Nightscoutを動かすことには成功しました。しかし
Chrome の拡張機能は「 この拡張機能は、セーフ ブラウジング保護強化機能で信頼されていません 」となっていて表示されません。
対応方法を教えて下さい。

**Comment by Navid200** [comment]
Why do you need Chrome extensions?
Please explain what you need it for.  There may be a solution.  But, I cannot look for it if I don't understand the need.  
I can use Chrome to see my readings in a browser.  Why would I need Chrome extensions?

**Reply by Nobukatsuwakui** [reply]
この拡張機能はNightscoutを表示させなくとも、Chrome を起動して何か作業している際にも、Chromeの上のバーに血糖値が小さく表示されるようになるというものです。画像の添付方法が判らないので言葉で書きました。いわゆる、M5 の様にChrome 上の表示させるもので、便利だったのです。

**Comment by Navid200** [comment]
Chrome 拡張機能が必要な理由
何のために必要なのか説明してください。 解決策があるかもしれません。 でも、必要性が分からないと探せません。
Chrome を使用して、ブラウザーで測定値を確認できます。 なぜ Chrome 拡張機能が必要なのですか?

**Comment by Navid200** [comment]
This is a translation of the second post by Nobukatsuwakui: 

This extension allows you to display your blood glucose level in a small bar above Chrome even when you're working on something with Chrome running, even if you don't want to display Nightscout. I don't know how to attach an image, so I wrote it in words. It was convenient because it was displayed on Chrome like the so-called M5.

**Comment by tzachi-dar** [comment]
What is the name of the extenation? Do we have the source code for it?

**Reply by Nobukatsuwakui** [reply]
拡張子とかわかりません。拡張機能に表示されている名前は：Nightscout Chrome Extension　です。Chrome WEB Store にあります。
そこからインストールするだけなので、ソースコードがどうなっているか、判りません。

**Reply by Navid200** [reply]
This is a translation using Google translate:

I don't know what the extension is. The name displayed in the extension is: Nightscout Chrome Extension. It's in the Chrome Web Store.
I just install it from there, so I don't know what the source code is.
